etna/datasets/tsdataset.py

```diff
@@ -900,21 +900,20 @@ def update_columns_from_pandas(self, df: pd.DataFrame, regressors: Optional[List
         if regressors is not None:
             self._regressors = list(set(self._regressors) | set(regressors))
 
-    def remove_columns(self, columns: List[str]):
+    def remove_features(self, features: List[str]):
         """Remove columns from the dataset.
 
         Columns that are not presented in the dataset will be ignored
```

Comment:
Should we somehow point out that after `make_future` columns will be added back?

Reply:
Add comments in the docstring